networking/external_dns_operator/nw-creating-dns-records-on-azure.adoc

```diff
@@ -8,4 +8,9 @@ toc::[]
 
 You can create DNS records on Azure by using the External DNS Operator.
 
+[IMPORTANT]
+====
+Using the External DNS Operator on a {entra-first}-enabled cluster or a cluster that runs in {azure-full} Government (MAG) regions or {entra-first} is not supported. 
```

Comment:
looks duplicated `{entra-first}` ?  The latter one can be removed.
```
Using the External DNS Operator on a {entra-first}-enabled cluster or a cluster that runs in {azure-full} Government (MAG) regions is not supported
```

Reply:
:heavy_check_mark: